tests/Resources/ODHVariables.resource

```diff
@@ -0,0 +1,3 @@
+//Put the ODH specific variables here
+*** Variables ***
+${ODH_DASHBOARD_TITLE}                                Open Data Hub Dashboard
```

Comment:
Can we use the same variable name for these variables that require different values? This could just be `${ODS_DASHBOARD_TITLE}` to make it simpler to override.